sdks/java/extensions/sql/build.gradle

```
@@ -62,7 +63,6 @@ dependencies {
   fmppTask "org.freemarker:freemarker:2.3.28"
   fmppTemplates library.java.vendored_calcite_1_20_0
   compile project(":sdks:java:core")
```

I wonder if the issue here is that :sdks:java:core should use the "shadow" configuration, like elsewhere.

```suggestion
  compile project(":sdks:java:core", configuration: "shadow")
```